jellyfin_kodi/entrypoint/service.py

```diff
@@ -249,11 +249,6 @@ def onNotification(self, sender, method, data):
             self.connect.remove_server(data['Id'])
             xbmc.executebuiltin("Container.Refresh")
 
-        elif method == 'UpdateServer':
```

Comment:
I'm not sure we should remove this entirely.  We need to provide a method for updating user passwords that doesn't involve resetting the local database

Reply:
You can't update the users password currently from the settings page can you?

Reply:
Thought you could, but apparently not.  That's something we'll need to add at some point.